manifests/fcontext.pp

```diff
@@ -44,25 +44,29 @@
 # @param restorecond_recurse Run restorecon recursive?
 #
 define selinux::fcontext (
-  $pathname,
-  $destination         = undef,
-  $context             = undef,
-  $filetype            = false,
-  $filemode            = 'a',
-  $equals              = false,
-  $restorecond         = true,
-  $restorecond_path    = undef,
-  $restorecond_recurse = false,
+  String $pathname,
+  Enum['absent', 'present'] $ensure  = 'present',
+  Optional[String] $destination      = undef,
+  Optional[String] $context          = undef,
+  Boolean $filetype                  = false, # ignored, 
+  Optional[String] $user             = undef,
+  Optional[String] $filemode         = 'a',
+  Boolean $equals                    = false,
+  Boolean $restorecond               = true,
```

Comment:
If we are already changing things, `restorcond` with d referes to the daemon. the utility is only called `restorecon`. The parameter only runs  `restorcon` and does nothing with `restorcond`.